core/templates/components/state-directives/outcome-editor/outcome-editor.component.html

```diff
@@ -23,6 +27,19 @@
   .oppia-incorrect-text {
     color: rgb(243, 13, 13);
   }
+
+  @media screen and (max-width: 500px) {
+    .oppia-mobile {
+      display: block;
+    }
+
+    .oppia-desktop {
+      display: none;
+    }
```

Comment:
Where are these used? They should be scoped.

Reply:
They are not used, removed them from the code.